src/main/java/mil/dds/anet/beans/Person.java

```diff
@@ -226,6 +235,19 @@ public void setPreviousPositions(List<PersonPositionHistory> previousPositions)
     return AnetObjectEngine.getInstance().getReportDao().search(query);
   }
 
+  @GraphQLQuery(name = "avatar")
+  public String getAvatar() throws IOException {
```

Comment:
It would be very neat if we can pass the radius as an optional parameter. 
This way, when rendering a single person, we could fetch the avatar as 200x200 image, but when rendering a graph of many people, we can request 50x50 images

Reply:
Sounds like a good idea, I will look into it.

Reply:
I suggest that we stick to radius sizes such as 1/2, 1/4, 1/8... 
The options then are:
- decode, sample, re-encode - relatively fast, I suggest we try to do this
- skip the decoding and encoding parts, and use DCT (https://libjpeg-turbo.org/About/SmartScale).  It may require more work to find the right library in java, but it could be very fast.

Reply:
Given the complexity, I think this one should be handled in its own issue

Reply:
Well, I also implemented this one after all.